_emails/us/california/san_luis_obispo_county.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "San Luis Obispo County, CA"
+permalink: "/san-luis-obispo"
```

Comment:
```suggestion
permalink: "/san-luis-obispo-county"
```

There is already `/sanluisobispo`, let's make sure they know it's the county

Reply:
ok definitely! thank you for pointing this out!